vite-plugin-ssr/node/client/router.ts

```diff
@@ -1,10 +1,11 @@
 export { navigate }
 
 import { assertWarning } from '../../utils/assert'
+import type { navigate as navigateOriginal } from '../../client/router'
```

Comment:
The path `../../client/router` does not exist in the `dist` directory and is therefore incorrect when consuming vite-plugin-ssr. See https://www.jsdelivr.com/package/npm/vite-plugin-ssr?path=dist%2Fcjs

Reply:
@brillout a lot of problems seem to stem from the fact that you bundle cjs + esm. is there a particular reason why you compile node-related code to cjs?

Reply:
Also, v0.4.45 does not include this fix. The package.json does not define types for the export "./client/router".

Reply:
Indeed. Fixed (76f1e75280f115c7ff67813ea0566254958ee3c8). It should now work with `v0.4.46`. Let me know if it doesn't.

Reply:
@vchirikov PR or reproduction welcome.